movie/src/main/res/layout/activity_home.xml

```diff
@@ -14,4 +14,18 @@
         app:defaultNavHost="true"
         app:navGraph="@navigation/nav_home" />
 
+    <TextView
+        android:id="@+id/no_internet_warning"
+        android:layout_width="match_parent"
+        android:layout_height="50dp"
```

Comment:
Nit: Please move the height to dimens.xml 

Also, is 50dp enough to show the view?